README.md

```diff
@@ -0,0 +1,8 @@
+ActiveModelValidators
+
+Additional validators: Email validator, Url validator, Immutability validator.
+
+Usage:
```

Comment:
please, describe it in more details. Also, add blocks as we have in  info_hub, custom_active_record_observer, etc